src/web-ui/src/public/components/DemoGuide.vue

```diff
@@ -0,0 +1,220 @@
+          {name : 'Real time events'}
+          ]
+        },
+        {
+          name: 'Amazon Pinpoint',
+          subsections: [
+            {name : 'Welcome Email'},
+            {name : 'Abandoned cart email'},
+            {name : 'Text alerts(SMS)'}
+          ]
+        },
+        {
+          name: 'Amazon Lex',
+          subsections: [
+            {name : 'Chatbot conversational interfaces'},
+          ]
+        }
+      ]
+    }
+  },
+  computed: {
+    showLevel: function() {
+      return this.showGuide || this.showGuideDetails;
+    }
+  },
+  methods: {
+    showDemoGuideLevel2: function() {
+      this.showGuide = !this.showGuide;
+      this.showUseCases = false;
+      this.showPersonalizeTopics = false;
+    },
+    showDemoGuideLevel3: function() {
+      this.showGuide = !this.showGuide;
+      this.showGuideDetails = !this.showGuideDetails;
+    },
+    hideDemoGuideLevel3: function() {
+      this.showGuide = !this.showGuide;
+      this.showGuideDetails = !this.showGuideDetails;
+      this.showUseCases = false;
+      this.showPersonalizeTopics = false;
+    },
+    showDemoGuideUseCases: function() {
+      this.showUseCases = true;
+    },
+    showDemoGuidePersonalizeTopics: function() {
+      this.showPersonalizeTopics = true;
+    },
+  },
+}
+</script>
+
+<style scoped>
+.demo-guide {
+  padding-top: 7rem;
+  font-weight: normal;
+  font-style: normal;
+  @font-face {
+  font-family: 'Amazon Ember';
+  src: local('../staticData/fonts/amazon-ember_rg.woff2'),url('../staticData/fonts/amazon-ember_rg.woff2'),
+  url('../staticData/fonts/amazon-ember_rg.woff');
+  }
+}
+.demo-guide-content {
+   background-color: #f8f9fa6b;
+   flex-shrink: 0;
+   padding: unset;
+}
+.demo-guide-level-1-text {
+   background-color: #152939;
+   flex-shrink: 0;
+   color: white;
+   justify-content: center;
+   display: block;
+   font-size: large;
+   padding: 1em;
+   border-top-left-radius: 10px;
+   border-top-right-radius: 10px;
+}
+.demo-guide-level-2-text {
+   background-color: #232f3e;
+   flex-shrink: 0;
+   color: white;
+   justify-content: center;
+   font-size: medium;
+   padding: 1em;
+}
+.demo-guide-level-3-text {
+   background-color: #232f3e;
+   flex-shrink: 0;
+   color: white;
+   font-size: small;
+   padding: 1em;
+   width: 100%;
+}
+.demo-guide-level-2{
+  text-align: center;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.demo-guide-use-cases{
+  border:solid;
+  border-width: thin;
+  width: 27em;
+  display: inline-block;
+  margin: 1em;
+}
+.modal {
+  width: 100%;
+  height: 100%;
+  background: #4a4b4cbd;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: block;
+  overflow-y: scroll;
+}
+.sidebar {
+  background-color: #31465f;
+  overflow-x: hidden;
+  box-sizing: border-box;
+}
+.icon {
+  width: 1em;
+  height: 1em;
+  fill: currentColor;
+}
+.color-amazon-orange {
+  color:#e88b01
+}
+.no-padding {
+   padding: 0;
+}
+.no-margin {
+  margin: 0 ;
+}
```

Comment:
Consider using Bootstrap `p-0` and `m-0` classes instead. Same comment applies to all other uses of `no-padding` and `no-margin` throughout PR.